modules/ossm-add-project-member-roll-resource-console.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_content-type: PROCEDURE
 [id="ossm-add-project-member-roll-recourse-console_{context}"]
-= Adding or removing projects from the service mesh using ServiceMeshMemberRoll resource with the web console
+= Adding or removing projects from the mesh using the member roll with the web console
 
-You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
+You can add or remove projects from the {SMProductShortName} using the `ServiceMeshMemberRoll` resource from the {product-title} web console. You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll` resource.
```

Comment:
@luksa looks like this comment was added more than once. I've made the change, but wanted to flag this in case my eyes are playing tricks on me and I've missed something.

I made the same change to the related procedure that uses the CLI. I'm not sure what else this comment might apply to.